packages/care-ops-auth/providers/workos.js

```diff
@@ -74,8 +74,10 @@ export class WorkosAuthProvider extends AuthProvider {
 
     if (pathName === AuthProvider.PATH_LOGOUT) {
       this.token = null;
```

Comment:
@paulfalgout this is what I'm saying though... the token gets cleared right here. So why can't we have an `if` condition to handle the "oops, already logged out" state:


```suggestion
      if (! this.token) {
          window.location.replace(AuthProvider.PATH_LOGIN);
          return;
      }
      this.token = null;
```

Reply:
Assuming the user is already logged out, why wouldn't this solve the original problem that was reported?

Reply:
It's possible for the token to be null here and still be logged in, and if you're actually logged in and we run window.locaton it just auto logs-in.